prow/cmd/prow-controller-manager/main.go

```diff
@@ -150,13 +150,28 @@ func main() {
 		logrus.WithError(err).Fatal("Error creating manager")
 	}
 
+	// The watch apimachinery doesn't support restarts, so just exit the
+	// binary if a build cluster can be connected later.
+	callBack := func() {
+		logrus.Info("Build cluster that failed to connect initially now worked, exiting to trigger a restart.")
+		interrupts.Terminate()
+	}
+
+	// We require operating on test pods in build clusters with the following
+	// verbs. This is used during startup to check that we have the necessary
+	// authorizations on build clusters.
+	requiredTestPodVerbs := []string{
+		"create",
+		"delete",
+		"list",
+		"watch",
+		"get",
```

Comment:
I don't believe "get" this is in the starter yaml files, and it took our controller out when I had to force upgrade today due to Github's RSA keys changing.

I simply changed my role to include it, but it might be worth going through all the yaml files and include it, if it's missing.